packages/kbn-test/src/functional_tests/cli/run_tests_cli.js

```diff
@@ -0,0 +1,63 @@
+import getopts from 'getopts';
+import { runTests } from '../../';
+import {
+  FUNCTIONAL_CONFIG_PATH,
+  API_CONFIG_PATH,
+} from '../../functional_tests/lib';
+
+const defaultConfigPaths = [FUNCTIONAL_CONFIG_PATH, API_CONFIG_PATH];
+
+/**
+ * Run servers and tests for each config
+ * Only cares about --config option. Other options
+ * are passed directly to functional_test_runner, such as
+ * --bail, --verbose, etc.
+ * In the future, this method will accept --tests-only,
+ * --servers-only.
+ * @param {configPaths}     array of paths to configs
```

Comment:
__issue:__ I've noticed that you use variable names in curly braces instead of type, that doesn't look like a valid JSDoc, shouldn't it be like this instead?

```js
@param {string[]} configPaths Array of paths to configs
```

iirc descriptions starts from capital letter to make it easier to distinguish variable name and description.

Reply:
While we're here, don't forget to include add your package to `.eslintrc.js` :)

Reply:
🤦‍♂️ thank you